apps/wallet-mobile/src/features/Portfolio/useCases/PortfolioTokenDetails/PortfolioTokenDetailsScreen.tsx

```diff
@@ -72,37 +71,38 @@ export const PortfolioTokenDetailsScreen = () => {
   }, [activeTab, setActiveTab, strings.overview, strings.performance, strings.transactions, styles.tab, styles.tabs])
 
   return (
-    <PortfolioTokenDetailLayout
-      topContent={
-        <Animated.View style={[styles.tabsSticky, isStickyTab ? styles.tabsStickyActive : styles.tabsStickyInactive]}>
-          {renderTabs}
-        </Animated.View>
-      }
-      onScroll={onScroll}
-      ListHeaderComponent={
-        <>
-          <Animated.View style={styles.header}>
-            <Spacer height={16} />
-
-            <PortfolioTokenBalance />
-
-            <Spacer height={16} />
-
-            <PortfolioTokenChart />
-
-            <Spacer height={16} />
-          </Animated.View>
-
-          <Animated.View>{renderTabs}</Animated.View>
-
-          <PortfolioTokenInfo />
-        </>
-      }
-      ListFooterComponent={loadingView}
-      {...getSectionListProps}
-    >
+    <TxFilter tokenId={tokenId}>
```

Comment:
entry point of the navigation (if the navigator doesn't have the safe-area-view) it is needed to be wrapped within it, also needs the background-colour cuz otherwise on iOS it will be transparent and the navigation will show the "stack" behind the screen.